assets/src/domain/eventEditor/ui/tickets/hooks/useEditTicketModal.ts

```diff
@@ -19,7 +24,13 @@ const useEditTicketModal: FormModal = ({ entityId, entityDbId }) => {
 
 	const onSubmit = useCallback<FormProps<UpdateTicketInput>['onSubmit']>(
 		(fields): void => {
-			updateEntity(fields);
+			const priceHasChanged = fields.price !== ticket?.price;
+			// if price has changed, reverseCalculate must be enabled
+			const reverseCalculate = priceHasChanged ? true : fields?.reverseCalculate;
+			updateEntity({ ...fields, reverseCalculate });
+			if (priceHasChanged) {
```

Comment:
👍

Reply:
agree, this reads so easy

Reply:
`if (fields.price !== ticket?.price)` - would make no sense to me